presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5970,6 +5970,7 @@ public void testTry()
     @Test
     public void TestTryExpression()
     {
+        // Tests from TestExpressionCompiler.java
```

Comment:
Don't put comment like `Tests from TestExpressionCompiler.java`, instead, merge them into `testTry` and explain in the comments that these tests doesn't have FROM clauses and might be constant folded.

Reply:
Can't reply to folded comments so I reply to them here (same topic :D )

I removed `SELECT 42 + TRY(ABS(-9223372036854775807 - 1))` because we already have `SELECT 123 + TRY(ABS(-9223372036854775807 - 1))`. I'll add them if it is necessary :)